client/gutenberg/extensions/tiled-gallery/editor.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import '@babel/polyfill';
```

Comment:
Editor itself already has `wp-polyfill` loaded so this is kinda not needed — should we leave it out and add `wp-polyfill` to dependencies for `editor.js`?

Reply:
Yes, I'd say so.